src/extension.ts

```diff
@@ -212,7 +214,7 @@ export async function activate(context: vscode.ExtensionContext) {
 	
 	return {
 		getStashedContext() : vscode.ExtensionContext {
-			return stashedContext;
+			return stashedContext;Will initialize Telemetry
```

Comment:
is that a comment?

Reply:
that's a bad copy when I accidently click on the mouse scroll wheel. i will remove it